manim/animation/movement.py

```diff
@@ -110,6 +110,21 @@ def interpolate_mobject(self, alpha: float) -> None:
 
 
 class MoveAlongPath(Animation):
+    """Make Mobject m1 move along path of Mobject m2.
```

Comment:
Comment: usually you want class/method docstrings to _not_ mention parameters by name.

```suggestion
    """Make one mobject move along the path of another mobject.
```